plugins/node/opentelemetry-instrumentation-ioredis/src/utils.ts

```diff
@@ -34,7 +34,9 @@ export const endSpan = (
 export const defaultDbStatementSerializer: DbStatementSerializer = (
   cmdName,
   cmdArgs
-) =>
-  Array.isArray(cmdArgs) && cmdArgs.length
-    ? `${cmdName} ${cmdArgs.join(' ')}`
-    : cmdName;
+) => {
+  if (Array.isArray(cmdArgs) && cmdArgs.length) {
+    return `${cmdName} ${cmdArgs[0]}`;
+  }
+  return cmdName;
+};
```

Comment:
please also update [the README](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-ioredis#custom-dbstatement-serializer)